ci/deploy_app.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+REGION="${REGION:=europe-west2}"
+CPU="${CPU:=4}"
+MEMORY="${MEMORY:=4G}"
+CONCURRENCY="${CONCURRENCY:=80}"
+CDN_URL="${CDN_URL:=https://cdn.census.gov.uk}"
+NEW_RELIC_APP_NAME="${NEW_RELIC_APP_NAME:=}"
+NEW_RELIC_LICENSE_KEY="${NEW_RELIC_LICENSE_KEY:=}"
+COOKIE_SETTINGS_URL="${COOKIE_SETTINGS_URL:=#}"
+ADDRESS_LOOKUP_API_URL="${ADDRESS_LOOKUP_API_URL:=#}"
+ADDRESS_LOOKUP_API_AUTH_ENABLED="${ADDRESS_LOOKUP_API_AUTH_ENABLED:=False}"
+ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS="${ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS:=10}"
+WEB_SERVER_TYPE="${WEB_SERVER_TYPE:=gunicorn-threads}"
+WEB_SERVER_WORKERS="${WEB_SERVER_WORKERS:=7}"
+WEB_SERVER_THREADS="${WEB_SERVER_THREADS:=10}"
+WEB_SERVER_UWSGI_ASYNC_CORES="${WEB_SERVER_UWSGI_ASYNC_CORES:=10}"
```

Comment:
These four `WEB_SERVER_` variables and `HTTP_KEEP_ALIVE` should be defined before runners environment variables.

Reply:
Moved them to the top, thanks!